vertical-pod-autoscaler/pkg/updater/eviction/pods_eviction_restriction.go

```diff
@@ -113,9 +134,9 @@ func (e *podsEvictionRestrictionImpl) Evict(podToEvict *apiv1.Pod, eventRecorder
 			Name:      podToEvict.Name,
 		},
 	}
-	err := e.client.CoreV1().Pods(podToEvict.Namespace).Evict(eviction)
+	err := e.client.CoreV1().Pods(eviction.Namespace).Evict(eviction)
```

Comment:
This seems unrelated (and does not change the code behaviour).

Reply:
Done.